core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/LogicalOp.scala

```diff
@@ -182,6 +200,22 @@ trait StateTransferFunc
     new Type(value = classOf[TablesPlotOpDesc], name = "TablesPlot"),
     new Type(value = classOf[JavaUDFOpDesc], name = "JavaUDF"),
     new Type(value = classOf[SortOpDesc], name = "Sort"),
+    new Type(value = classOf[SklearnLROp], name = "LR"),
+    new Type(value = classOf[SklearnLRCVOp], name = "LRCV"),
+    new Type(value = classOf[SklearnRROp], name = "RR"),
+    new Type(value = classOf[SklearnRRCVOp], name = "RRCV"),
+    new Type(value = classOf[SklearnSDGOp], name = "SDG"),
```

Comment:
please follow the naming convention: we call them `xxxOpDesc`.

Reply:
Also, please make the "name" attribute more meaningful. For example, it is very hard to know what does "PA" stands for.
A complete example:
```
new Type(value = classOf[SklearnSupportVectorMachineOpDesc], name = "SklearnSupportVectorMachine")
```
Don't worry that it will be long, in the future we may introduce other SVM from other libraries, it is necessary to differentiate them.

Reply:
Fixed.